consensus/bor/heimdallgrpc/client.go

```diff
@@ -41,12 +52,26 @@ func NewHeimdallGRPCClient(address string) *HeimdallGRPCClient {
 	log.Info("Connected to Heimdall gRPC server", "address", address)
 
 	return &HeimdallGRPCClient{
-		conn:   conn,
-		client: proto.NewHeimdallClient(conn),
+		conn:                  conn,
+		client:                protoV1.NewHeimdallClient(conn),
+		borQueryClient:        borTypes.NewQueryClient(conn),
+		checkpointQueryClient: checkpointTypes.NewQueryClient(conn),
+		clerkQueryClient:      clerkTypes.NewQueryClient(conn),
+		milestoneQueryClient:  milestoneTypes.NewQueryClient(conn),
 	}
 }
 
 func (h *HeimdallGRPCClient) Close() {
 	log.Debug("Shutdown detected, Closing Heimdall gRPC client")
 	h.conn.Close()
 }
+
+// removePrefix removes the http:// or https:// prefix from the address, if present.
+func removePrefix(address string) string {
+	if strings.HasPrefix(address, "http://") || strings.HasPrefix(address, "https://") {
+		return address[strings.Index(address, "//")+2:]
+	}
+	return address
+}
+
+var defaultTimeout = 1 * time.Second
```

Comment:
Shall we place this def on top of the file?

Reply:
Done.